modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view_table.jsp

```diff
@@ -0,0 +1,239 @@
+					HashMapBuilder.<String, Object>put(
+						"actions", StringUtil.merge(kbAdminManagementToolbarDisplayContext.getAvailableActions(kbArticle))
+					).build());
+
+				row.setPrimaryKey(String.valueOf(kbArticle.getResourcePrimKey()));
+				%>
+
+				<liferay-ui:search-container-column-text
+					cssClass="table-cell-expand table-cell-minw-200"
+					name="title"
+				>
+					<clay:content-row>
+						<clay:content-col>
+							<span class="sticker sticker-document sticker-rounded sticker-secondary">
+								<clay:icon
+									symbol="document-text"
+								/>
+							</span>
+						</clay:content-col>
+
+						<clay:content-col
+							expand="<%= true %>"
+						>
+
+							<%
+							PortletURL viewURL = kbArticleURLHelper.createViewWithRedirectURL(kbArticle, currentURL);
+							%>
+
+							<aui:a href="<%= viewURL.toString() %>">
+								<%= HtmlUtil.escape(kbArticle.getTitle()) %>
+							</aui:a>
+						</clay:content-col>
+					</clay:content-row>
+				</liferay-ui:search-container-column-text>
+
+				<liferay-ui:search-container-column-text
+					align="right"
+					name="num-of-kb-folders"
+					value="--"
+				/>
+
+				<liferay-ui:search-container-column-text
+					align="right"
+					name="num-of-kb-articles"
+				>
+					<%= kbArticleViewDisplayContext.getChildKBArticlesCount(scopeGroupId, kbArticle) %>
+				</liferay-ui:search-container-column-text>
+
+				<liferay-ui:search-container-column-text
+					align="right"
+					name="view"
+				>
+					<%= kbArticle.getViewCount() %>
+				</liferay-ui:search-container-column-text>
+
+				<liferay-ui:search-container-column-text
+					align="right"
+					name="modified-date"
+				>
+					<span class="text-default">
+						<liferay-ui:message arguments="<%= kbArticleViewDisplayContext.getModifiedDateDescription(kbArticle) %>" key="x-ago" />
+					</span>
+				</liferay-ui:search-container-column-text>
+
+				<liferay-ui:search-container-column-text
+					align="right"
+					name="status"
+				>
+					<aui:workflow-status helpMessage='<%= kbArticle.isExpired() ? dateFormatDateTime.format(kbArticle.getExpirationDate()) : "" %>' markupView="lexicon" showHelpMessage="<%= kbArticle.isExpired() %>" showIcon="<%= false %>" showLabel="<%= false %>" status="<%= kbArticle.getStatus() %>" />
+				</liferay-ui:search-container-column-text>
+
+				<liferay-ui:search-container-column-text
+					align="right"
+				>
+					<clay:dropdown-actions
+						aria-label='<%= LanguageUtil.get(request, "show-actions") %>'
+						dropdownItems="<%= kbArticleViewDisplayContext.getKBArticleDropdownItems(kbArticle) %>"
+						propsTransformer="admin/js/KBDropdownPropsTransformer"
+					/>
+				</liferay-ui:search-container-column-text>
+			</c:otherwise>
+		</c:choose>
+	</liferay-ui:search-container-row>
+
+	<liferay-ui:search-iterator
+		displayStyle="list"
+		markupView="lexicon"
+		resultRowSplitter="<%= kbFolderView ? new KBResultRowSplitter() : null %>"
+		searchContainer="<%= searchContainer %>"
```

Comment:
Is this attribute `searchContainer` needed here? the descriptive view does not have it. 

Also, I was thinking.... what is best here: should we move the <liferay-ui:search-container> tag to the view.jsp, as it is common to both views (`view_table.jsp` and _view_descriptive.jsp_) and simplify them a little bit? What have backends to say here, any recommendations?

Reply:
about the `searchContainer`:
It seems that only is used on some cases but I'm not sure if is needed (I found this https://issues.liferay.com/browse/LPS-177195 looking on it) so I can not answer that

about the liferay-ui:search-container, IMO it is more clean to have this on the views jsps not in the main, but I do not know if affect putting it on the main.